backend/tests/manual_upload_test.py

```
@@ -0,0 +1,395 @@
+
+    # Create output directory
+    os.makedirs(args.output_dir, exist_ok=True)
+
+    # Initialize tester
+    tester = UploadTester(args.server_url)
+
+    # Test server connectivity first
+    if not tester.test_server_connectivity():
+        return 1
+
+    print(f"\nStarting upload validation tests...")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)